frontend/src/components/Pages/FrontPage/RobotCards/PressureStatusView.tsx

```diff
@@ -12,21 +17,55 @@ const StyledTopography = styled(Typography)<{ $fontSize?: 24 | 16 | 18 | 32 | 40
     font-size: ${(props) => props.$fontSize};
 `
 export interface PressureStatusViewProps {
-    pressure?: number
+    pressureInBar?: number
+    pressureStatus?: PressureStatus
     itemSize?: 24 | 16 | 18 | 32 | 40 | 48 | undefined
+    upperPressureWarningThreshold: number
+    lowerPressureWarningThreshold: number
 }
 
-const PressureStatusView = ({ pressure, itemSize }: PressureStatusViewProps): JSX.Element => {
+const PressureStatusView = ({
+    pressureInBar,
+    pressureStatus,
+    itemSize,
+    upperPressureWarningThreshold,
+    lowerPressureWarningThreshold,
+}: PressureStatusViewProps): JSX.Element => {
     const barToMillibar = 1000
     let icon_color: string = tokens.colors.interactive.primary__resting.hex
 
-    if (!pressure) {
+    if (!pressureInBar || !upperPressureWarningThreshold || !lowerPressureWarningThreshold) {
+        pressureStatus = PressureStatus.Error
```

Comment:
EX will never have a value for pressure cause it doesn't need pressure. Setting the status to error just because the value is mission is misleading